pwiz_tools/Skyline/Model/Databinding/Entities/ColumnToolTips.resx

```diff
@@ -1269,6 +1269,9 @@ higher than the other extent.</value>
   <data name="InChiKey" xml:space="preserve">
     <value>Standardized molecular identifier</value>
   </data>
+  <data name="KEGG" xml:space="preserve">
+    <value>Standardized molecular identifier</value>
+  </data>
   <data name="SMILES" xml:space="preserve">
     <value>Standardized molecular identifier</value>

```

Comment:
Maybe we should mention that this stands for "simplified molecular-input line-entry system" (which coincidentally has the same first letters as "Standardized molecular identifier").